course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,257 @@
+# RQM, ecologized
+
+<!-- vim-markdown-toc GFM -->
+* [On the necessary changes to the RQM syllabus](#on-the-necessary-changes-to-the-rqm-syllabus)
+* [Draft of proposed schedule](#draft-of-proposed-schedule)
+    * [Week 1](#week-1)
+    * [Week 2](#week-2)
+    * [Week 3](#week-3)
+    * [Week 4](#week-4)
+    * [Week 5](#week-5)
+    * [Week 6](#week-6)
+    * [Week 7](#week-7)
+    * [Week 8](#week-8)
+    * [Week 9](#week-9)
+    * [Week 10](#week-10)
+    * [Week 11](#week-11)
+    * [Week 12](#week-12)
+    * [Possible additional ideas](#possible-additional-ideas)
+* [Labs and lectures from Martin](#labs-and-lectures-from-martin)
+    * [Structured population dynamics](#structured-population-dynamics)
+    * [Predator prey population dynamics](#predator-prey-population-dynamics)
+    * [Lectures](#lectures)
+* [The original RQM course](#the-original-rqm-course)
+    * [Lecture topics](#lecture-topics)
+    * [Discussions](#discussions)
+    * [Exercises](#exercises)
+
+<!-- vim-markdown-toc -->
+
+## On the necessary changes to the RQM syllabus
+- To tailor the course for ecology students, we need to remove some of the existing material.
+- I have tried to summarize the differences between the original schedule and the schedule I propose further down. I did this by classifying all the lectures and tutorials into the overarching themes of the RQM course, and quantified the number of classes dedicated to a theme in the original and proposed schedules (original -> proposed). This can serve as a rough overview for how the course direction is changing:
+    - 13 -> 8.5 Reproducibility (includes all R classes)
+    - 5 -> 1.5 Authorship, collaboration and publishing
+    - 6 -> 0 Openness in data and science
+    - 5 -> 8 Project work
+    - 0 -> 6 Theoretical ecology
+- Most of the topics in Christie's course involve both practical and theoretical aspects. For example, all of the R-coding would fall under reproducibility, but there are also discussions dedicated to reproducibility.
+- Note that there are 2 or 3 of these topics per week in the original RQM schedule. Time-wise, this means that a topic approximately equals one class (lecture or tutorial). There were 29 topics in the original RQM course, which we have reduced to 24 topics to fit our course. The total lecture time is still the same, just that each topic gets more attention.
+
+## Draft of proposed schedule
```

Comment:
We still need to go through the content week by week and see if it is realistic to fit into its given time span.